Prototope/Layer+Layout.swift

```diff
@@ -66,6 +66,30 @@ extension Layer {
 	}
 	
 	
+	/** Moves the receiver so that its left side is aligned with the left side of its parent layer. */
+	public func moveToLeftSideOfParentLayer(margin: Double = 0.0) {
+		if let parent = self.parent {
+			self.frameMinX = -margin
```

Comment:
I don’t mind changing it though, just trying to keep related things consistent :)